deploy/v2/ansible/roles/os-disk-setup/tasks/installation_path_prechecks.yml

```
@@ -0,0 +1,15 @@
+---
+
+# Check if installation paths are correct. 
+- name: Pre-check installation path
+  stat:
+    path: "{{ item.value.mount_point }}"
+  loop: "{{ disk_dict|dict2items }}"
+  register: path_not_exist
+
+# If any one of paths is incorrect, path_unready_status_flag will be defined.
+- name: Get intallation path status
+  set_fact:
+     path_unready_status_flag: "{{ item.stat.exists}}"
```

Also nit suggestion:
```suggestion
     path_status_flag: "{{ item.stat.exists}}"
```

---

Good catch thanks!